docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -30,7 +30,15 @@ read or write (or no) access to project settings, logs, task
 annotations, and more. These can be requested for specific members or
 for entire guilds. The project admins (or Evergreen admins, if no
 project admins exist) are then able to approve/reject requested access
-for their project.
+for their project. 
+
+In order to access this functionality, you must look up the the MANA resource 
```

Comment:
I'd add a mention that you have to click that little filter button in mana to look up by resource type (cause it was hard for us to see as well haha)